.github/workflows/ci.yml

```
@@ -0,0 +1,56 @@
+name: GitHub CI
+
+on:
+  push:
+  pull_request:
+    paths-ignore:
+      - 'README.md'
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [Windows-latest, Ubuntu-latest, macOS-latest]
+        configuration: [Debug, Release]
+        include:
+          - os: windows-latest
+            cmake-parameters: -G "Visual Studio 16 2019" -A x64
+            folder: win64
+          - os: ubuntu-latest
+            cmake-parameters: -G "Unix Makefiles"
+            folder: linux64
+          - os: macos-latest
+            cmake-parameters: -G "Xcode"
```

You only need quotes if it has spaces, but they can be left there for consistency.
```suggestion
            cmake-parameters: -G Xcode
```